config/README.md

```diff
@@ -0,0 +1,125 @@
+# Rustic Configuration Specification
+
+Rustic is a backup tool that allows users to define their backup options using a TOML configuration file. The configuration file consists of various sections and attributes that control the behavior of Rustic for different commands and sources.
+
+This specification covers all the available sections and attributes in the Rustic configuration file. Users can customize their backup behavior by modifying these attributes according to their needs.
+
+## Merge Precedence
+
+The merge precedence for values is:
+
+    Commandline Arguments >> Environment Variables >> Configuration File
+
+Values parsed from the `configuration file` can be overwritten by `environment variables`, which can be overwritten by `commandline arguments` options. Therefore `commandline arguments` have the highest precedence.
+
+**NOTE**: There are the following restrictions:
+
+- You can overwrite values, but for most values, you cannot "unset" them on a higher priority level.
+
+- For some integer values, you cannot even overwrite with the value `0`, e.g. `keep-weekly = 5` in the `[forget]` section of the config file cannot be overwritten by `--keep-weekly 5`.
+This is not relevant for env variables, only for some values available in the `config` and as `CLI` option.
+
+## Sections and Attributes
+
+### Global Options
+
+| Attribute         | Description                                            | Default Value  | Example Value | Environment Variable |
+|-------------------|--------------------------------------------------------|----------------|---------------|---------------|
+| dry-run           | If true, performs a dry run without making any changes. | false       || RUSTIC_DRY_RUN |
+| log-level         | Logging level. Possible values: "off", "error", "warn", "info", "debug", "trace". | "debug" || RUSTIC_LOG_LEVEL |
+| log-file          | Path to the log file.                                 | TODO | "/log/rustic.log" | RUSTIC_LOG_FILE |
+| no-progress       | If true, disables progress indicators.               | false          || RUSTIC_NO_PROGRESS |
+| progress-interval | The interval at which progress indicators are shown. | "100ms"        || RUSTIC_PROGRESS_INTERVAL |
+| use-profile       | An array of profiles to use.                           | Empty array    || RUSTIC_USE_PROFILE |
+
+### Repository Options
+
+| Attribute         | Description                                            | Default Value  | Example Value | Environment Variable |
+|-------------------|--------------------------------------------------------|----------------|---------------|---------------|
+| cache-dir         | Path to the cache directory.                          | TODO        | Default cache dir, e.g., ~/.cache/rustic | RUSTIC_CACHE_DIR |
```

Comment:
yes, exactly like it is written. But I think it will be the repository-depending dir, i.e. `~/.cache/rustic/$REPO_ID`, and when you specify another one it won't add the repository id, IIRC.